src/app/workflow/workflow.component.ts

```diff
@@ -253,6 +278,9 @@ export class WorkflowComponent extends Entry implements AfterViewInit {
       }
       this.setUpWorkflow(workflow);
     });
+    this.tokenQuery.tokens$.subscribe((tokens: Token[]) => {
+      this.setTokens(tokens);
```

Comment:
I don't think you need to save the tokens; just set zenodoAccountIsLinked here.

Reply:
done